website/api/controllers/try-fleet/view-sandbox-or-redirect.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+
+
+  friendlyName: 'View sandbox',
+
+
+  description: 'Display "Sandbox" page or redirect users to their Fleet Sandbox instance.',
```

Comment:
This friendly name, filename, and description could use some help.  Because of the nature of the login trick, being extra explicit here will be helpful.  Especially since the form post kinda "feels" like a redirect.  Also, this is not either displaying a page or redirecting users to their Fleet Sandbox instance:  it's either (A) displaying a brief interstitial form post that autosubmits to a subdomain and returns HTML which then sets browser localstorage to contain credentials and consider the user logged in, or (B) redirecting because the user's Fleet Sandbox instance is expired.

So how about:

- filename: `view-sandbox-teleporter-or-redirect-because-expired.js`
- `friendlyName: 'View sandbox teleporter or redirect because expired'`
- `description: 'Display "Sandbox teleporter" page (an auto-submitting interstitial HTML form used as a hack to grab a bit of HTML from the Fleet Sandbox instance, which sets browser localstorage to consider this user logged in and "teleports" them, magically authenticated, into their Fleet Sandbox instance running on a different domain), or redirect the user to a page about their sandbox instance being expired.'`
- `moreInfoUrl: 'https://github.com/fleetdm/fleet/pull/6380'`